### ISSUE TITLE
paper or blogpost

### ISSUE THREAD

**Issue by dereyly** [ultralytics/yolov5#4]
Hello
Thank you for this project.
Do you have paper or blogpost how you can achieve so amazing results?

**Comment by glenn-jocher** [comment]
@dereyly we would like to publish a paper later on in the year, but at the moment we are still evolving and researching these models, so they are not complete yet. 

We recently updated the wiki with a custom training tutorial though, this is the best documentation for the moment on how to use the repo.

**Comment by glenn-jocher** [comment]
@dereyly and all others regarding publication and naming:

Thank you for your feedback! Regarding publication, we would very much like to be able to publish a paper detailing the various modifications employed to achieve these results (among all 3 major components: architecture, loss function and training methodology), however we are extremely limited on resources, and thus must smartly select how best to deploy these in order to keep our business viable as a going concern, while also continuing our work of pushing the boundaries of what's possible in this and other fields. 

Importantly **these models are neither static nor complete** at this time. Our recent open-sourcing of this work is simply part of our ongoing research, and is not any sort of final product, and for this reason it is not accompanied by any publication. Our current goal is to continue internal R&D throughout the remainder of 2020, and hopefully open source and publish at least a short synopsis of this to Arxiv by the end of the year. 

We have published several peer reviewed [works](https://www.ultralytics.com/publications) prior as a company, and myself personally as well ([Google Scholar](https://scholar.google.com/citations?hl=en&user=swSrGtsAAAAJ)), mostly in particle physics and antineutrino detection. Our most notable publication was the world's first [Antineutrino Global Map](https://www.nature.com/articles/srep13945), published in Nature, Scientific Reports as part of ongoing work Ultralytics performed for the U.S. National Geospatial-Intelligence Agency.

It was our work reconstructing neutrino events which led us to develop an ancillary interest in AI, and in this field we are pursuing research that is both transparent and reproducible, which builds on years of hard work (by myself and many community contributors) porting and perfecting YOLOv3 to PyTorch from Darknet, which pjreddie amazingly began and @alexeyab so impressively continued and advanced. Most importantly in following the spirit of YOLO, we are targeting results that we hope may be **in reach of every individual** who would express an interest in the field, not merely available to those with unlimited resources at their disposal. Our smallest model, to cite an example, trains on COCO in only 3 days on one 2080Ti, and runs inference faster and more accurately than EfficientDet D0, which was trained on 32 TPUv3 cores by the Google Brain team. By extension we aim to comparably exceed D1, D2 etc. with the rest of the YOLOv5 family.

Regarding naming, we do take note of the comments surrounding the issue. YOLOv5 is an internal designation assigned to this work, which is now open-sourced. The exact name employed here is not a concern for us (we are open to alternatives!), we are instead focused on producing, improving, and delivering results to our clients, and to the open-source community by extension when our contract terms allow for it, which we push for often.

We appreciate all feedback, and we will try to be as responsive as we can going forward!

**Comment by leokwu** [comment]
Great sharing. Look forward to the paper discussion.
